LOG:  database system was shut down at 2017-07-01 11:18:29 PDT
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
FATAL:  role "dannytsoi" does not exist
FATAL:  database "/Users/dannytsoi/Documents/dev_Stuff/TEST-ENV/django-test/postg" does not exist
FATAL:  database "/Users/dannytsoi/Documents/dev_Stuff/TEST-ENV/django-test/danny" does not exist
FATAL:  database "/Users/dannytsoi/Documents/dev_Stuff/TEST-ENV/django-test/postg" does not exist
